### PR TITLE
Add _class and _while keywords to magik lexer

### PR DESCRIPTION
**Pull request by StevenLooman**
Add `_class` and `_while` keywords to magik lexer. Newer versions of the magik language support these new keywords.

### CODE REVIEW

**Comment by pyrmont** [comment]
@StevenLooman Could you add an example to the visual sample to confirm this is working?

**Comment by StevenLooman** [comment]
Good point, added.

File: spec/visual/samples/magik [hunk 1]
```diff
@@ -8,7 +8,8 @@ def_slotted_exemplar(
         :test_object,
         {
                 {:slot, _unset}
-        })
+        },
+        {_class |java.lang.Integer|)
```
Comment (pyrmont) [inline spec/visual/samples/magik]: It's not causing a lexing problem but is there supposed to be a closing brace in the class declaration?

Comment (StevenLooman) [reply spec/visual/samples/magik]: Nice catch! Fixed.

**Comment by StevenLooman** [comment]
Thank you.

**Comment by pyrmont** [comment]
@StevenLooman No worries! Thanks for contributing the PR :)

**Pull request merged**